sidecar_acceptor_test.go

```diff
@@ -102,9 +80,23 @@ func TestRegisterSidecar(t *testing.T) {
 				},
 			},
 		},
-		expectedErr: "signature not valid for public key",
+		expectedErr: "error verifying sidecar offer",
```

Comment:
We should actually check that the error contains `signature not valid for public key` here.